.github/workflows/npm_publish.yaml

```diff
@@ -0,0 +1,31 @@
+name: npm publish
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Install Node
+        uses: actions/setup-node@v4
+        with:
+          node-version: '20.x'
+          registry-url: 'https://registry.npmjs.org'
+          always-auth: true
+
+      - name: Install Depencencies
+        run: yarn install
+
+      - name: Build, Test, and Package
+        run: yarn prepack && yarn test
+
+      - name: Publish npm package to both places
+        run: |
+          yarn publish --access public
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
```

Comment:
Do we actually care about publishing this package if it's only used internally for our docs?

Reply:
yeah, our docs are currently pulling in the `@metamask/open-rpc-docs-react` package. Until now, I've hacked together some customizations using a `.patch` file. So, we're forking/publishing our own so it's not so "fragile."